man/tpm2_rsaencrypt.1.md

```diff
@@ -41,7 +41,9 @@ The key referenced by keyHandle is **required** to be:
 
```

Comment:
I really think Github should have a way to comment on commit messages, so I'll comment here:

You need to add a Fixes: #608

Reply:
Ah, nevermind. I see that you are adding the tag in the next commit so feel free to ignore this comment.

Reply:
Yes becuase #508 had both -I and -o modifications, so I don't close it until all the work is done.